### PR TITLE
LPS-126596 Update variable names

### PR DESCRIPTION
**Pull request by csierra**
/cc @martamedio

### CODE REVIEW

File: modules/apps/oauth2-provider/oauth2-provider-api/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -4,6 +4,7 @@ CLIENT_CREDENTIALS=Client Credentials
 REFRESH_TOKEN=Refresh Token
```
Comment (liferay-appsec) [inline modules/apps/oauth2-provider/oauth2-provider-api/src/main/resources/content/Language.properties:4]: Check that added properties are either titles or sentences

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/edit_application_left_column.jsp [hunk 2]
```diff
@@ -115,6 +115,9 @@ OAuth2Application oAuth2Application = oAuth2AdminPortletDisplayContext.getOAuth2
 
 						<div class="allowedGrantType <%= cssClassesStr %>">
 							<c:choose>
+								<c:when test="<%= grantType.equals(GrantType.AUTHORIZATION_CODE) || grantType.equals(GrantType.AUTHORIZATION_CODE_PKCE) %>">
+									<aui:input checked="<%= checked %>" data="<%= data %>" label="<%= grantType.name() %>" name="<%= name %>" onchange='<%= liferayPortletResponse.getNamespace() + "updateTrustedApplicationSection();" %>' type="checkbox" />
```
Comment (liferay-appsec) [inline modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/edit_application_left_column.jsp:119]: This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation:

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/edit_application_left_column.jsp [hunk 3]
```diff
@@ -221,6 +224,19 @@ OAuth2Application oAuth2Application = oAuth2AdminPortletDisplayContext.getOAuth2
...
 		</c:if>
+
+		<c:if test="<%= oAuth2AdminPortletDisplayContext.hasAddTrustedApplicationPermission() %>">
+			<clay:col
+				id='<%= liferayPortletResponse.getNamespace() + "trustedApplicationSection" %>'
+				lg="12"
+			>
+				<h3 class="sheet-subtitle"><liferay-ui:message key="trusted-application" /></h3>
+
+				<aui:field-wrapper>
```
Comment (liferay-appsec) [inline modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/META-INF/resources/admin/edit_application_left_column.jsp:235]: This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation:

File: modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/content/Language.properties [hunk 4]
```diff
@@ -97,6 +97,8 @@ this-application-wants-the-following-permissions=This application wants the foll
 this-operation-cannot-be-performed-because-you-cannot-impersonate-x=This operation cannot be performed because you cannot impersonate user {0}.
```
Comment (liferay-appsec) [inline modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/content/Language.properties:97]: Check that added properties are either titles or sentences

**Review by liferay-appsec — COMMENTED** [review]
This is an automatic review suggestion

**Comment by csierra** [comment]
ci:forward

**Comment by csierra** [comment]
ci:forward

**Comment by csierra** [comment]
ci:forward

**Comment by martamedio** [comment]
ci:forward

**Comment by martamedio** [comment]
ci:stop

**Comment by martamedio** [comment]
Please, see: https://github.com/liferay-appsec/liferay-portal/pull/285

**Pull request closed without merging**